pkgs/by-name/el/element-desktop/keytar/default.nix

```
@@ -10,11 +10,27 @@
   xcbuild,
   fetchNpmDeps,
   npmHooks,
+  electron,
+  buildPackages,
+  runCommand,
 }:
 
 let
   pinData = lib.importJSON ./pin.json;
 
+  # Make sure the native modules are built against electron's nodejs version
```

```suggestion
  # Make sure the native modules are built against electron's ABI
```
(or something similar)
the headers are not only different because they use a different nodejs internally, but e.g because they replace certain nodejs v8 headers with chromium v8 headers.